packages/components/src/Button/Button.css

```diff
@@ -177,11 +176,13 @@ a.button.disabled:hover {
 .base {
   min-height: calc(var(--base-unit) * 2.25);
   padding: 0 calc(var(--space-base) - 4px);
+  letter-spacing: var(--typography--letterSpacing-loose);
```

Comment:
I think we can reduce the amount of CSS here if we leave the `letterSpacing` declaration as-is on line 20 and set `letter-spacing: 0` on the small button class on line 173